src/providers/translations/__tests__/TranslationsProvider.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  render,
+  within,
+} from '@testing-library/react';
+import { Alert } from '../../../components/Alert';
+import TranslationsProvider from '../TranslationsProvider';
+
+describe('rendering', () => {
+  it.each([
```

Comment:
This shouldn't be `it.each`. It should looks as `GlobalPropsProvider.test.jsx`.

Reply:
fixed

I also added test for nested translations and fixed what was broken.